Extension/package.json

```diff
@@ -42,6 +42,168 @@
+          },
+          {
+            "id": "open.project",
+            "title": "Open your project folder",
+            "description": "Open an existing folder that contains your C++ project, or create a new folder. \n[Pick a folder](command:workbench.action.files.openFolder)",
+            "when": "workspaceFolderCount == 0",
+            "doneOn": {
+              "command": "workbench.action.files.openFolder"
+            },
+            "media": {
+              "altText": "A screenshot that shows a C++ project opened in the file Explorer.",
+              "path": "walkthrough/cpp-folder-explorer.png"
+            }
+          },
+          {
+            "id": "setup.intellisense",
+            "title": "Configure C++ IntelliSense",
+            "description": "Verify important IntelliSense settings like Compiler Path, Compiler Args, IntelliSense Mode, and Include Path. \n[Open IntelliSense Configuration](command:C_Cpp.ConfigurationEditUI?-2)",
+            "doneOn": {
+              "command": "C_Cpp.ConfigurationEditUI"
+            },
+            "media": {
+              "altText": "A screenshot that shows the IntelliSense Configurations UI with important settings highlighted.",
+              "path": {
+                "light": "walkthrough/intellisense-config-light.png",
+                "dark": "walkthrough/intellisense-config-dark.png",
+                "hc": "walkthrough/intellisense-config-hc.png"
+              }
+            }
+          },
+          {
+            "id": "build.project.mac",
+            "title": "Build your C++ file",
+            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-clang-mac#_build-helloworldcpp)",
+            "when": "isMac",
+            "media": {
+              "altText": "A markdown file with instructions for configuring the default build task on Mac with Clang.",
+              "path": "walkthrough/build-project-mac.md"
+            }
+          },
+          {
+            "id": "build.project.linux",
+            "title": "Build your C++ file",
+            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-linux#_build-helloworldcpp)",
+            "when": "isLinux",
+            "media": {
+              "altText": "A markdown file with instructions for configuring the default build task on Linux with GCC.",
+              "path": "walkthrough/build-project-linux.md"
+            }
+          },
+          {
+            "id": "build.project.windows",
+            "title": "Build your C++ file",
+            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-msvc#_build-helloworldcpp)",
+            "when": "isWindows",
+            "media": {
+              "altText": "A markdown file with instructions for configuring the default build task on MSVC with Windows.",
```

Comment:
For "configuring the default build task on MSVC with Windows" does it mean to say "build task on Windows with MSVC"?

Reply:
Good catch :)

Reply:
Fixed